lisa/sut_orchestrator/azure/tools.py

```diff
@@ -55,3 +58,66 @@ def get_generation(self) -> str:
         else:
             generation = "1"
         return generation
+
+
+class LisDriver(Tool):
+    """
+    This is a virtual tool to detect/install LIS drivers.
```

Comment:
Readers might want to know what the acronym stands for (if there is literature to link here, the better :))